test/boost/view_complex_test.cc

```diff
@@ -840,7 +840,7 @@ void test_commutative_row_deletion(cql_test_env& e, std::function<void()>&& mayb
         }});
     });
 
-    e.local_db().get_compaction_manager().perform_major_compaction(&e.local_db().find_column_family("ks", "vcf")).get();
+    e.local_db().get_compaction_manager().perform_major_compaction(e.local_db().find_column_family("ks", "vcf").as_table_state()).get();
```

Comment:
why can't the test use table::compact_all_sstables()?

Reply:
it's not testing the compaction_manager...

Reply:
Looks like an improvement for later. Tried to minimize the changes needed as much as possible.